bin/cli.js

```diff
@@ -77,7 +77,7 @@ const saveSession = Boolean(sessionTimeout)
 const sessionFile = path.resolve(os.tmpdir(), 'bitwarden-session.txt')
 
 const dmenuArgsParsed = dmenuArgs ? dmenuArgs.split(/\s+/) : []
-const dmenuPswdArgsParsed = ['-p', 'Password:', '-nf', 'black', '-nb', 'black'].concat(
+const dmenuPswdArgsParsed = ['-p', 'Password:', '-P'].concat(
```

Comment:
I ran your changes locally, and it gives an error.
```
 andrew  bitwarden-dmenu-pr  ./bin/cli.js        
node:events:356
      throw er; // Unhandled 'error' event
      ^

Error: write EPIPE
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:775:11)
    at Socket._write (node:net:787:8)
    at writeOrBuffer (node:internal/streams/writable:400:12)
    at _write (node:internal/streams/writable:341:10)
    at Socket.Writable.write (node:internal/streams/writable:345:10)
    at /home/andrew/Code/development/bitwarden-dmenu-pr/src/executable-wrappers/dmenu.js:16:17
    at new Promise (<anonymous>)
    at /home/andrew/Code/development/bitwarden-dmenu-pr/src/executable-wrappers/dmenu.js:11:3
Emitted 'error' event on Socket instance at:
    at emitErrorNT (node:internal/streams/destroy:188:8)
    at emitErrorCloseNT (node:internal/streams/destroy:153:3)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {
  errno: -32,
  code: 'EPIPE',
  syscall: 'write'
}
```

Im not sure what version of dmenu you are using, but mine does not include a `-P` flag (`dmenu-5.0` for reference)

Reply:
That's because I have the [password](https://tools.suckless.org/dmenu/patches/password/) patch installed. But I reverted that line back to what it was and instead added a command to README for people who want to use that patch.